app/javascript/packages/captcha-submit-button/captcha-submit-button-element.ts

```diff
@@ -55,14 +55,14 @@ class CaptchaSubmitButtonElement extends HTMLElement {
     this.#onReady(async () => {
       const { recaptchaSiteKey: siteKey, recaptchaAction: action } = this;
 
-      let token;
+      let token: string | undefined;
```

Comment:
If this TypeScript type were assigned previously, it'd also have been flagged as an issue:

```
app/javascript/packages/captcha-submit-button/captcha-submit-button-element.ts:65:7 - error TS2322: Type 'string | undefined' is not assignable to type 'string'.
  Type 'undefined' is not assignable to type 'string'.

65       this.tokenInput.value = token;
         ~~~~~~~~~~~~~~~~~~~~~
```